src/algorithm/centroid.rs

```diff
@@ -107,22 +120,28 @@ mod test {
     /// Tests: Centroid of LineString
     #[test]
     fn empty_linestring_test() {
-        let vec = Vec::new();
-        let linestring = LineString(vec);
+        let vec : Vec<Point<f64>> = Vec::new();
+        let linestring : LineString<f64> = LineString(vec);
         let centroid = linestring.centroid();
         assert!(centroid.is_none());
     }
     #[test]
     fn linestring_one_point_test() {
         let p = Point::new(40.02f64, 116.34);
-        let linestring = LineString(vec![p]);
+//<<<<<<< HEAD
+        let mut vect : Vec<Point<f64>> = Vec::new();
+        vect.push(p);
+        let linestring : LineString<f64> = LineString(vect);
+//=======
+//        let linestring = LineString(vec![p]);
+//>>>>>>> faa0840be34d10713206e403c8956ca17a0a26b4
```

Comment:
This is failed automatic merge annotation. That's disturbing at the minimum. And I didn't see that it was commented, thus thought that it will not compile.